server/routers/interview.ts

```diff
@@ -118,6 +118,26 @@ export const interviewRouter = router({
 
         return interview;
       }),
+    manyByIds: protectedProcedure
```

Comment:
`forExport` might be a better name, since this is a very specific use-case.

Reply:
Done ✅